h2/src/main/org/h2/mvstore/tx/TransactionStore.java

```diff
@@ -348,23 +420,28 @@ public void logUndo(Transaction t, long logId) {
      *
      * @param t the transaction
      * @param maxLogId the last log id
-     * @param oldStatus last status
+     * @param hasChanges false for R/O tx
```

Comment:
“R/O tx” means read-only transaction? But it's not clear what is read-only in this context. Transaction from read-only storage, transaction that was explicitly set to be read-only (how?), something else? I saw in tests that this probably means transaction without changes so name of the parameter is better and more correct than it's documentation.